### ISSUE TITLE
How to disable cluster.

### ISSUE THREAD

**Issue by kuzarandom** [pavel-corsaghin/react-native-leaflet#31]

**Comment by miallo** [comment]
I don't think this is supported right now - [in the original (unmaintained) repo](https://github.com/reggie3/react-native-webview-leaflet/blob/master/HTML/src/MapMarkers.tsx#L38) from which [the html in this fork generated](https://github.com/pavel-corsaghin/react-native-leaflet/blob/main/android/src/main/assets/leaflet.html) the `useMarkerClustering` is by default set to true, but this prop is not forwarded anywhere.

Right now in my company we are drastically simplifying this library and throwing things like the marker clustering out, because we don't need it and it caused us issues anyways... @orangecms or I can ping you once we have it in a working shape and published it if you want.

**Comment by johnsan96** [comment]
oh really? Im currently working on a project that needs a leaflet map with clustering. Will I able to use the clustering when its thrown out?

**Comment by miallo** [comment]
@johnsan96 So far the idea was not to change this project, but to create another one that is drastically stripped down (also concerning features) to make the maintenance easier. So if this library works fine for you, there is absolutely no reason to change it and you can continue using it ;)

**Comment by caydey** [comment]
If your looking for a quick "hacky" fix, what I done was modified `android/src/main/assets/leaflet.html` by replacing `maxClusterRadius:80` with `maxClusterRadius:-1`
